genienlp/tasks/almond.py

```diff
@@ -217,7 +217,7 @@ def _make_example(self, parts, dir_name=None, **kwargs):
             _id, sentence, target_code = parts[:3]
         else:
             _id, sentence, target_code = parts
-        question = 'translate from english to thingtalk'
+        question = ''
```

Comment:
If we ever care about multi-task learning, this will be a problem.

Reply:
Having a fixed question doesn't enable multitask learning either. We would need to set the question field as a function of other example fields.

Reply:
It's fixed per task. So it would be a different question with different tasks. That's how decanlp did it anyway (and T5 too, IIRC)

Reply:
OK, I'm reverting this change. `--overwrite_question ""` should work just fine.